filter-org-members-action/action.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Filter Organization Members Action'
+description: 'Get list of users belong to a specific organization from an input user list.'
+inputs:
+  token:
+    required: true
+  organization:
+    required: true
+  usernames:
+    required: true
+runs:
+  using: 'node16'
```

Comment:
Suggest using 20 instead? It should be LTS.

Reply:
Refined.